test_podcastparser.py

```diff
@@ -21,8 +21,11 @@
 import os
 import glob
 import json
+from StringIO import StringIO
```

Comment:
This fails for Python 3: https://ci.appveyor.com/project/thp/podcastparser/build/1.0.13/job/xansm34033i7juu6

Use this instead:

    try:
        from io import StringIO
    except ImportError:
        from StringIO import StringIO